internal/builder/config.go

```diff
@@ -64,7 +64,6 @@ type Module struct {
 	Import string `mapstructure:"import"` // if not specified, this is the path part of the go mods
 	GoMod  string `mapstructure:"gomod"`  // a gomod-compatible spec for the module
 	Path   string `mapstructure:"path"`   // an optional path to the local version of this module
-	Core   bool   `mapstructure:"core"`   // whether this module comes from core, meaning that no further dependencies will be added
```

Comment:
I would say that we should keep this property for a couple of releases, telling users what to use instead.